client/app/storage.service.ts

```diff
@@ -29,7 +30,7 @@ export class StorageService {
    */
   public storedPaymentUuid$ = this.storedPaymentUuidSubject.asObservable();
 
-  private storedCurrentStepSubject: BehaviorSubject<number> = new BehaviorSubject<number>(
+  private storedCurrentStepSubject: BehaviorSubject<DemoProgress> = new BehaviorSubject<number>(
```

Comment:
Shouldn't this be
```suggestion
  private storedCurrentStepSubject: BehaviorSubject<DemoProgress> = new BehaviorSubject<DemoProgress>(
```

Reply:
Yeah there were a few more I missed, but ended up cleaning up downstream. Going to dismiss for now, since it gets fixed in a downstream change